lib/foods.js

```diff
@@ -67,7 +69,17 @@ $('document').ready(function() {
       event.preventDefault();
       this.blur();
     }
-
   })
 
+  function addFoodToTable(name, calories) {
+    var foodEntry = "<td contenteditable='true' class='food-name'>" + name + "</td>";
+    var caloriesEntry = "<td contenteditable='true' class='food-calories'>" + calories + "</td>";
+    var deleteButton = "<td class='food-delete'><button>-</button></td>";
+    var newRow = "<tr class='food-row'>" +
```

Comment:
i like that you've set variables for each of these. it makes it easier to read - i also like that you have a separate function to add the food to the table